pkg/controller/masteruserrecord/sync.go

```diff
@@ -0,0 +1,74 @@
+package masteruserrecord
+
+import (
+	"context"
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"reflect"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type Synchronizer struct {
+	hostClient    client.Client
+	memberClient  client.Client
+	memberUserAcc *toolchainv1alpha1.UserAccount
+	recordUserAcc toolchainv1alpha1.UserAccountEmbedded
+	record        *toolchainv1alpha1.MasterUserRecord
+}
+
+func (s *Synchronizer) synchronizeSpec() error {
+	if !reflect.DeepEqual(s.memberUserAcc.Spec, s.recordUserAcc.Spec) { // TODO maybe do not use reflect :-)
+		// when UserAccount spec in record is updated - is not same as in member
+		s.memberUserAcc.Spec = s.recordUserAcc.Spec
+		if err := updateStatus(s.hostClient, s.record, "updating", ""); err != nil {
+			return err
+		}
+		err := s.memberClient.Update(context.TODO(), s.memberUserAcc)
```

Comment:
Should be vice versa :) We want to copy the user account status to the master user record status. Not the other way around.